torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9974,6 +9989,14 @@ def ref_pairwise_distance(input1, input2):
            dtypes=floating_and_complex_types(),
            sample_inputs_func=sample_inputs_linalg_slogdet,
            decorators=[skipCUDAIfNoMagma, skipCUDAIfRocm, skipCPUIfNoLapack],),
+    OpInfo('linalg.trace',
+           aten_name='linalg_trace',
+           op=torch.linalg.trace,
+           supports_out=False,
```

Comment:
Doesn't it? I don't know what happened with this discussion https://github.com/pytorch/pytorch/pull/62714#discussion_r778830799

Reply:
The tests seem to fail if enabling supports_out. Perhaps this option is assuming that `out` is a positional argument?

Reply:
What's the error it throws? `out=` should be a kwonly argument, so it's not clear to me a priori what's the problem there...

Reply:
Ah, sorry! The test failure was due to the missing `check_scalar_type_device_layout_equal(result, self);`!